config-templates/ibm/ibm-vpc-block-csi-driver/5.0/deployment.yaml

```diff
@@ -0,0 +1,723 @@
+apiVersion: v1
+kind: List
+metadata:
+  name: ibm-vpc-block-csi-driver
+  namespace: kube-system
+  annotations:
+    version: 5.0
+    revision: 1
```

Comment:
@mssachan Please confirm if revision is required for new templates released

Reply:
its required